ansible/roles-infra/infra-osp-create-inventory/tasks/main.yml

```diff
@@ -44,7 +46,7 @@
       private_ip_address: "{{ server.private_v4 }}"
       public_ip_address: "{{ server.public_v4 | default('') }}"
       image_id: "{{ server.image.id | default('') }}"
-      ansible_ssh_extra_args: "-o StrictHostKeyChecking=no"
+      ansible_ssh_extra_args: "-o StrictHostKeyChecking=false"
```

Comment:
Should not change `no` to `false` in this case since it is within an SSH command line flag.

Reply:
Didn't realize, fixed.